13-networking-in-flutter/projects/final/lib/ui/main_screen.dart

```diff
@@ -96,27 +96,27 @@ class _MainScreenState extends State<MainScreen> {
           items: [
             BottomNavigationBarItem(
               icon: SvgPicture.asset(
-                'assets/images/icon_recipe.svg',
-                color: _selectedIndex == 0 ? green : Colors.grey,
-                semanticsLabel: 'Recipes',
+                  'assets/images/icon_recipe.svg',
+                  color: _selectedIndex == 0 ? green : Colors.grey ,
+                  semanticsLabel: 'Recipes'
               ),
-              title: Text('Recipes'),
+              label: 'Recipes'
             ),
             BottomNavigationBarItem(
-              icon: SvgPicture.asset(
-                'assets/images/icon_bookmarks.svg',
-                color: _selectedIndex == 1 ? green : Colors.grey,
-                semanticsLabel: 'Bookmarks',
-              ),
-              title: Text('Bookmarks'),
+                icon: SvgPicture.asset(
+                    'assets/images/icon_bookmarks.svg',
+                    color: _selectedIndex == 1 ? green : Colors.grey ,
+                    semanticsLabel: 'Bookmarks'
+                ),
+                label: 'Bookmarks'
```

Comment:
I'm not sure what is happening. The BottomNavigationBarItem label throws errors that's why I changed it to title.

Reply:
As of this commit everything compiles and works fine with me.